pkgs/tools/security/dirstalk/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
```

Comment:
[These](https://github.com/stefanoj3/dirstalk/blob/master/.goreleaser.yml#L24) ldflags are missing. Also, adding `-s` and `-w` would also be preferred.

Reply:
I will skip the "BuildTime" entry as it doesn't really add value.